vllm/model_executor/layers/quantization/fp8.py

```diff
@@ -469,7 +469,8 @@ def apply(self,
               renormalize: bool,
               use_grouped_topk: bool,
               topk_group: Optional[int] = None,
-              num_expert_group: Optional[int] = None) -> torch.Tensor:
+              num_expert_group: Optional[int] = None,
+              routing_func: Callable = torch.topk) -> torch.Tensor:
```

Comment:
Instead of setting the default to torch.topk everywhere, could we set it to None by default and only use it if it's not None? And call it `custom_routing_function` or something similar to account for it not being fused_topk or grouped topk?